FormalisingMathematics2025/Section03functions/Sheet1.lean

```diff
@@ -22,24 +22,19 @@ and satisfying some axioms, and so on.
 
 -- Lean is a dependently-typed language
 -- Every expression has a type, and `#check` can tell you the type
-
```

Comment:
These newline removals aren't important but this file is already pretty long imo and sticking comments clearly to the blocks they're describing feels a tiny bit better to me. This also matches how the rest of the file is laid out. Totally not critical though